osmnx/core.py

```diff
@@ -787,7 +787,11 @@ def add_edge_lengths(G):
 
     # first load all the edges' origin and destination coordinates as a
     # dataframe indexed by u, v, key
-    coords = np.array([[u, v, k, G.nodes[u]['y'], G.nodes[u]['x'], G.nodes[v]['y'], G.nodes[v]['x']] for u, v, k in G.edges(keys=True)])
+        try:
```

Comment:
This indentation error is causing the CI tests to fail.

Reply:
Apologies. It was a copy'n'paste error from emacs as I haven't got github integration sorted .